src/app/public/styles/_switch.scss

```diff
@@ -86,6 +86,7 @@ $sky-switch-size: 22px !default;
   line-height: $sky-switch-size;
   flex: 1 1 auto;
   width: 100%;
+  margin-right: 10px;
```

Comment:
Do we want to use a sky variable for this?

Reply:
Good call. Updated